src/main/java/net/mcreator/blockly/java/blocks/NumberFromTextBlock.java

```diff
@@ -38,8 +38,9 @@ public class NumberFromTextBlock implements IBlockGenerator {
 				num = element;
 		}
 		if (num != null) {
+			master.addTemplate("utils/double_from_text.java.ftl");

```

Comment:
When Java-based templates use FTL, there are two requirements:

1. _ prefix in the file, see templates for procedures
2. have fallback when ftl does not exist - see existing cases, e.g., PrintTextBlock